t04_refactored.py

```diff
@@ -231,9 +231,36 @@ def team_19_adv():
 ###################################################################################
 
 def team_20_adv():
-    pass
-    # TODO Add your code here
+    """
+    https://docs.google.com/document/d/1aO8daq4WLqOsChlhvmClBHNZEx4faJcmsvGXPyOU990/edit?usp=sharing
+    :return:
+    America Gaona Borges
+    Jessica Douthit
+    """
+    global isDead
+    input("Go with him further Up, or Down? ")
+    if input == "Up":
```

Comment:
can't use the variable `input` because it overwrites the method `input` that is part of python. Use the `direction` variable I suggested in the previous comment